projects/ngx-cookiebot/src/lib/ngx-cookiebot.config.ts

```diff
@@ -3,6 +3,7 @@
  */
 export abstract class NgxCookiebotConfig {
   blockingMode: 'auto' | 'manual' | string;
+  consentMode?: string;
```

Comment:
Could we be more specific here? The documentation seems to indicate that `disabled` is the only valid value to pass, no?

Reply:
Yes, it makes sense to check the allowed values.